app/src/main/AndroidManifest.xml

```diff
@@ -19,22 +19,24 @@
 
         <activity
                 android:name=".ui.WireActivity"
+                android:exported="true"
                 android:screenOrientation="portrait"
                 android:theme="@style/AppTheme.SplashScreen"
-                android:windowSoftInputMode="adjustResize|stateHidden"
-                android:launchMode="singleTop"
-                android:exported="true">
+                android:windowSoftInputMode="adjustResize|stateHidden">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
-            <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
+
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="wire"
-                        android:host="access"/>
+
+                <data
+                        android:host="access"
+                        android:scheme="wire" />
+                <data
+                        android:host="teams.wire.com"
```

Comment:
Maybe we could move all urls to gradle - gradle variables can be injected into manifest and we could get rid of `ServerConfig` companion objects and use only `BuildConfig` ones - this way we would have only single source of url data instead of at least three that we have right now (`BuildConfig`, `ServerConfig`, manifest).
What do you guys think about that?

Reply:
Thanks for the comment, what I've got from your comment, that's you're pointing to two different subjects! one is injecting default settings for the serverConfig, but injecting deep links in the manifest via the Gradle is possible too. I'll address these two in other tasks under the quality ticket.